test/functional/unified-spec-runner/schema.ts

```diff
@@ -118,7 +130,41 @@ export interface Test {
 }
 export interface ExpectedEventsForClient {
   client: string;
-  events: ExpectedEvent[];
+  eventType?: string;
+  events: (ExpectedCommandEvent | ExpectedCmapEvent)[];
+}
+export interface ExpectedCommandEvent {
```

Comment:
since this is a duplicate of the original ExpectedEvent interface, should `ExpectedEvent` be deleted below? Or is it still used somewhere?

Reply:
`ExpectedEvent` now removed.